src/main.rs

```diff
@@ -50,6 +49,26 @@ macro_rules! plural {
     };
 }
 
+fn is_rept(node: &Node) -> bool {
+    matches!(node.type_data(), NodeType::Rept(..))
+}
```

Comment:
This is definitely a good candidate to move into rgbds-obj; maybe the following function too? (Along with similar functions for the other variants.)

Reply:
Done in https://github.com/gbdev/rgbds-obj/pull/18.